localstack/services/dynamodb/dynamodb_listener.py

```diff
@@ -393,7 +399,26 @@ def return_response(self, method, path, data, headers, response):
         if event_sources_or_streams_enabled and records and 'eventName' in records[0]:
             if 'TableName' in data:
                 records[0]['eventSourceARN'] = aws_stack.dynamodb_table_arn(table_name)
-
+            # StreamViewType determines what information is written to the stream for the table
```

Comment:
nitpick: I'd suggest moving this logic into a small util function, to prevent the handler code in this method to become too long/convoluted.